### PR TITLE
Login and Sign up screen

### PR DESCRIPTION
**Pull request by samuldecosta**
Updates 
1. Login and Sign up screen .
2. Route to onboarding after login or signup.
3. Swap component name EducationCertificate <=> levelOfComfort.
4. updated schema for education field.
5. Change checkbox to radio button for relocation field.

### CODE REVIEW

File: src/components/atoms/FormCustomRadioGroup.tsx [hunk 1]
```diff
@@ -5,33 +5,57 @@ import {
...
+import RadioButtonUncheckedIcon from "@mui/icons-material/RadioButtonUnchecked";
+import RadioButtonCheckedIcon from "@mui/icons-material/RadioButtonChecked";
 import styled from "@emotion/styled";
 import theme from "src/theme";
 import { IFormField } from "types";
 import { useController, UseFormReturn } from "react-hook-form";
 import { StyledLabel } from "./common";
 
-const StyledFormControlLabel = styled(FormControlLabel)<{ selected: boolean }>`
+const StyledFormControlLabel = styled(FormControlLabel)<{
```
Comment (sandeep-asan) [inline src/components/atoms/FormCustomRadioGroup.tsx:18]: don't override the style keep the default style of MUI for radio

File: src/components/atoms/FormInputWithIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import { FunctionComponent, useState } from "react";
```
Comment (sandeep-asan) [inline src/components/atoms/FormInputWithIcon.tsx:1]: don't need this component. instead use FormTextField and add additional optional props endAdornment like startAdornment.

General TextField accept  a variety of icons. If we put the icons logic inside this component, we will have to define many checks here.
Based on `Separation Of Concerns `principle, I recommend you separate icons logic from this component like the startAdornment

File: src/components/molecules/OnboardingDesktopTimeline.tsx [hunk 3]
```diff
@@ -16,9 +16,13 @@ export const OnboardingDesktopTimeline = ({
   return (
     <>
       <Box sx={{ w: "200px", p: 2 }}>
-        <Box sx={{ display: "flex", justifyContent: "space-between" }}>
-          <Typography>Progress</Typography>
-          <Typography>
+        <Box
+          sx={{ display: "flex", justifyContent: "space-between", mb: "7px" }}
```
Comment (sandeep-asan) [inline src/components/molecules/OnboardingDesktopTimeline.tsx:20]: spacing(margin/padding) should be multiple of 4. like 4px 8px 12px 16px etc

File: src/components/organisms/SingUp.tsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+          </Typography>
+          <Typography sx={{ fontSize: "18px" }}>
+            Lorem ipsum dolor sit amet
+          </Typography>
+        </Box>
+        <Box>
+          <Button
+            variant="outlined"
+            sx={{ padding: "8px 10px", borderRadius: "30px", mb: "30px" }}
+            startIcon={<GoogleIcon />}
```
Comment (sandeep-asan) [inline src/components/organisms/SingUp.tsx:56]: use the google icon from figma

File: src/components/atoms/FormCustomRadioGroup.tsx (hunk 1 above)
Comment (samuldecosta) [reply src/components/atoms/FormCustomRadioGroup.tsx]: removed styles and varient from component

File: src/components/organisms/SingUp.tsx (hunk 4 above)
Comment (samuldecosta) [reply src/components/organisms/SingUp.tsx]: updated icon

**Pull request merged**